docs/autogenerated-menu/New Content/3 career/people_ops/talent/recruitment_process_overview.md

```diff
@@ -0,0 +1,156 @@
+
+---
+
+## Applicant Tracking
+
+We use **Workable**, an applicant tracking system, to manage every stage of the hiring process. Workable helps us:
+- Keep track of open roles and candidate progress.
+- Store CVs, interview notes, and communications.
+- Ensure timely updates and a smooth experience for candidates.
+
+By keeping Workable accurate and up to date, we provide **clarity and efficiency**, making sure that candidates don’t fall through the cracks.
+
+---
+
+## The Early Stages: Finding and Assessing Candidates
+
+### Demand: What, When, and How Many
+
+We need to be **clear and consistent** about what roles we’re hiring for and why. Hiring demand at Burendo comes from two key sources:
+
+- **Tactical (Immediate) Demand** – When customer engagements require rapid staffing.
+- **Strategic (Forecast) Demand** – When we plan ahead to grow our team and expand capabilities.
+
+The talent team needs **time to do their work effectively**. Frequent last-minute changes make it harder to find and hire the best people.
+
+### Sourcing: How We Find Great People
+
+We find candidates in four main ways:
+
+1. **Direct Public Profile** – Some candidates come to us directly via our website or at industry events.
+2. **Referral (Staff Introduction)** – Great people know great people. If a team member introduces someone we hire, we reward them.
+3. **Talent Team (Active Searching)** – Our internal recruiters proactively look for the best talent.
+4. **Agency (Chosen Partners)** – We work with a select group of recruiters who understand what we need.
+
+No matter where a candidate comes from, they all go through the same **firm but fair** interview process. Recruitment is a **two-way street**—candidates must like us just as much as we like them.
+
+---
+
+## CV Review: The First Filter
+
+Every CV is different, but there are a few things we always check:
+- **Job Duration** – Have they changed jobs every 3–6 months? Have they stayed in one place for 10+ years? Both are worth asking about.
+- **Certificates vs. Experience** – Learning is great, but we want hands-on experience too.
+- **Details** – A candidate should be able to clearly explain what *they* did in previous roles, not just what their team achieved.
+- **Grammar & Consistency** – Spelling mistakes and vague job descriptions can indicate a lack of attention to detail.
+
+A great CV gives us confidence that this candidate is worth speaking to.
+
+---
+
+## The Initial Call: Getting to Know Each Other
+
+This is our **first real interaction** with the candidate, and it’s as much about them getting to know us as it is about us getting to know them. We cover:
+
+- **Past** – Our history, our roots, and how we got to where we are today.
+- **Present** – Who we work with, what we do, and how we operate.
+- **Future** – Where we’re heading and how they could be part of it.
+- **Human Connection** – Do they communicate well? Do they seem like a good cultural fit?

```

Comment:
I don't like culture fit.  We don't want more of the same people.  We want new people that make us better.
```suggestion
- **Human Connection** – Do they communicate well? Do they seem like someone who can improve/extend/expand our culture?

```